_health-care/_js/_form_progress.js

```diff
@@ -1,73 +1,63 @@
-      if ($(allTabs[i]).find('a').attr('aria-controls') === $(nextPanel).attr('id')) {
-        nextTab = allTabs[i];
-        break;
-      }
+// When a particular tab is selected, show the first set of questions in that topic
+function showSectionOfSelectedTab(activeTab) {
+  var activePanel = $('#' + $(activeTab).attr('aria-controls'));
+  var $activeSection = $(activePanel).find('.form-section').first();
+
+  $activeSection.show();
+}
+
+// Show that the tab of the current set of questions is selected
+function selectTabOfActiveSection(nextPanel) {
+  var $currentTab = $('.tab-title.active');
+  var allTabs = $('.tab-title');
+  var nextTab = '';
+
+  for (var i = 0; i < allTabs.length; i++) {
+    if ($(allTabs[i]).find('a').attr('aria-controls') === $(nextPanel).attr('id')) {
+      nextTab = allTabs[i];
+      break;
     }
+  }
 
-    $currentTab.removeClass('active');
-    $currentTab.find('a').attr('aria-selected', 'false');
-
-    $(nextTab).addClass('active');
-    $(nextTab).find('a').attr('aria-selected', 'true');
-  };
+  $currentTab.removeClass('active');
+  $currentTab.find('a').attr('aria-selected', 'false');
 
-  // Show the next set of questions
-  var showNextSection = function() {
-    var $currentSection = $('.form-section:visible');
-    var indexOfCurrentSection = $allFormSections.index($currentSection);
-    var $currentPanel = $currentSection.parents('.content');
-    var nextSection = $allFormSections[indexOfCurrentSection + 1];
-    var $nextPanel = $(nextSection).parents('.content');
+  $(nextTab).addClass('active');
+  $(nextTab).find('a').attr('aria-selected', 'true');
+}
 
-    $currentSection.hide();
+// Show the next set of questions
+function showNextSection()  {
+  let $allFormSections = $('.form-section');
+  var $currentSection = $('.form-section:visible');
+  var indexOfCurrentSection = $allFormSections.index($currentSection);
+  var $currentPanel = $currentSection.parents('.content');
+  var nextSection = $allFormSections[indexOfCurrentSection + 1];
+  var $nextPanel = $(nextSection).parents('.content');
 
-    if ($nextPanel.attr('id') !== $currentPanel.attr('id')) {
-      $currentPanel.removeClass('active');
-      $currentPanel.attr('aria-hidden', 'true');
+  $currentSection.hide();
 
-      $nextPanel.addClass('active');
-      $nextPanel.attr('aria-hidden', 'fasle');
+  if ($nextPanel.attr('id') !== $currentPanel.attr('id')) {
+    $currentPanel.removeClass('active');
+    $currentPanel.attr('aria-hidden', 'true');
 
-      selectTabOfActiveSection($nextPanel);
-    }
+    $nextPanel.addClass('active');
+    $nextPanel.attr('aria-hidden', 'fasle');
 
-    $(nextSection).show();
-  };
+    selectTabOfActiveSection($nextPanel);
+  }
 
-  var initForm = function() {
-    $('.content.active').find('.form-section').first().show();
+  $(nextSection).show();
+};
 
-    $('#continueButton, #skipButton').on('click', function() {
-      showNextSection();
-    });
+export function init() {
```

Comment:
Only in ES6!!!

It replaces the whole module pattern hack we had been using previously. See

http://blog.teamtreehouse.com/get-started-ecmascript-6